src/queries/courtConfig.js

```diff
@@ -0,0 +1,40 @@
+import gql from 'graphql-tag'
+
+export const CourtConfig = gql`
```

Comment:
one thing that facu mentioned in one of my pr's is to just query what we need not everything

Reply:
I think that eventually we will use all this data but will remove the ones we don't currently use and if needed let them be added later 👍